gui/src/components/s3/s3.json

```diff
@@ -156,6 +158,8 @@
       "account_id":"Account id",
       "canonical_id":"Canonical id",
       "reset-password": "Reset password",
+      "update-password": "Update password",
```

Comment:
Please check if this is change password or update password and update accordingly.

Reply:
Its Update only.